src/platform/startup/store.js

```diff
@@ -37,18 +38,22 @@ export const commonReducer = {
  * sets up the Redux devtools in development and adds redux-thunk as middleware.
  *
  * @param {Object} [appReducer={}] An object with reducer functions as properties
+ * @param {Array} A list of analytics events to capture when redux actions are fired
```

Comment:
I think it's expecting that the first word after the types of your params is the argument name, e.g.
`* @param {Array} AnalaticsEvents A list of analytics....`